<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #AAAAFF, stop:1 #ffaaff);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="barraTitulo">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="menuBtn">
        <property name="text">
         <string>  Menu</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>menu.png</normaloff>menu.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>938</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="minimizar">
        <property name="styleSheet">
         <string notr="true">background-color:transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>blackLine.png</normaloff>blackLine.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="maximizar">
        <property name="styleSheet">
         <string notr="true">background-color:transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>maximizar.png</normaloff>maximizar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cerrar">
        <property name="styleSheet">
         <string notr="true">background-color:transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>cross.png</normaloff>cross.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="inferior_barraT">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="menu_barra">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
background-color: rgb(255, 170, 255);
}

QPushButton{
border-radius:20px;
font:75 12pt &quot;Arial Narrow&quot;;
text-align:left;
}

QPushButton:hover{
background-color:purple;
border-top-left-radius:20px;
border-bottom-left-radius:20px;
font:75 12pt &quot;Arial Narrow&quot;;
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>40</number>
         </property>
         <item>
          <widget class="QPushButton" name="ingredientes">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align:left;</string>
           </property>
           <property name="text">
            <string>  Ingredientes</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ingredients.png</normaloff>ingredients.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="recetas">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>  Recetas</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>recipe-book.png</normaloff>recipe-book.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>  Pedidos</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>orders.png</normaloff>orders.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>  Historial de ventas</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>history.png</normaloff>history.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>  Cuenta</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>account.png</normaloff>account.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>324</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Bakeeper</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="contenedorPaginas">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <widget class="QWidget" name="page1">
            <property name="styleSheet">
             <string notr="true">background-color:transparent;</string>
            </property>
            <widget class="QTableWidget" name="tableWidget">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>100</y>
               <width>781</width>
               <height>481</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius:5px;</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="searchLineEdit">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>20</y>
               <width>751</width>
               <height>22</height>
              </rect>
             </property>
             <property name="placeholderText">
              <string>Ingresa el nombre del ingrediente</string>
             </property>
            </widget>
            <widget class="QPushButton" name="printButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>900</x>
               <y>100</y>
               <width>93</width>
               <height>28</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 0, 255);</string>
             </property>
             <property name="text">
              <string>Imprimir</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page2">
            <property name="styleSheet">
             <string notr="true">background-color:purple;</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
